### DISCUSSION TITLE
Multiple User Supports

### DISCUSSION THREAD

**Discussion by rockylca2014** [shirokumacode/overwatch-omnic-rewards#3] [Ideas]
I wonder whether by just using one instance of the program it would be possible to do multiple account or it would just be more simple to just run multiple instance of the program

**Comment by shirokumacode** [comment]
I initially designed the app to use one account to simplify. Right now it would be too much work for me atm to implement multi-account in the app and would introduce complexity and possible bugs. But I know what I need to change in order to do so. When I have more time I would definitively like to work on this. 

**If more people request this I will make this a priority.**

Meanwhile you can run multiple instances of the app, making sure they are in different directories (different configuration files). 

If you are proficient in Python, I would recommend you check out [utils/example.py](https://github.com/shirokumacode/overwatch-omnic-rewards/blob/main/utils/example.py). It should be fairly easy to adapt the script to make it multi-account, however be mindful the example doesn't have any error handling.
